src/KJ/Magento/Command/Order/Create/DummyCommand.php

```diff
@@ -40,6 +40,9 @@ protected function configure()
             ->addOption('product', null, InputOption::VALUE_OPTIONAL, "A product SKU to use for the order")
             ->addOption('store', null, InputOption::VALUE_OPTIONAL, "A store ID to use for the order")
             ->addOption('shipping', null, InputOption::VALUE_OPTIONAL, "A shipping method code to use for the order")
+			->addOption('custom_options', null, InputOption::VALUE_OPTIONAL, "Custom options to use for the product, if applicable")
```

Comment:
@wsheldon looks like there's an indentation / spacing issue here and throughout the PR.